### PR TITLE
Add md5 of repositories to TMX filename (fixes #841)

### LINKED ISSUE #841: TMX download broken when trying to download two different products
I set the TMX download page to en-US, Normal, Central and download the TMX.

If I then set the same option but for mozilla.org, I get the same TMX generated the first time instead of the new one.

I haven't used the download page in a while, not sure when it broke.

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by flodolo** [comment]
Looks like Bugzilla is having some issues, the usual query in mozilla-l10n-query failed last night, and it failed for me to in the browser.

Let's see if Travis passes this time.

File: app/models/tmx_downloading.php [hunk 1]
```diff
@@ -47,16 +49,13 @@ function ($e) {
...
-    $content = TMX::create($strings, $locale, 'en-US');
-    $target_file = 'mozilla_en-US_' . $locale . '.tmx';
-} else {
-    $content = TMX::createOmegat($strings, $locale, 'en-US');
-    $target_file = 'mozilla_en-US_' . $locale . '_forOmegaT.tmx';
-}
-
-$target_file_path = WEB_ROOT . 'download/' . $target_file;
+$tmx_id = md5($requested_repositories);
+$target_file_name = "mozilla_en-US_{$locale}_{$tmx_id}_{$check['tmx_format']}.tmx";
```
Comment (TheoChevalier) [inline app/models/tmx_downloading.php]: Wondering if this is introducing a XSS vector, are we making sure `$check['tmx_format']` can’t contain anything else than "normal" or "omegat"?

Comment (flodolo) [inline app/models/tmx_downloading.php]: We are: https://github.com/mozfr/transvision/blob/master/app/inc/tmx_options.php#L4-L10

Comment (TheoChevalier) [inline app/models/tmx_downloading.php]: Good! Thanks for checking

**Pull request merged**